ci/scripts/run-e2e-test.sh

```diff
@@ -81,6 +81,16 @@ RUST_BACKTRACE=1 target/debug/risingwave_e2e_extended_mode_test --host 127.0.0.1
 echo "--- Kill cluster"
 cargo make ci-kill
 
+echo "--- e2e, ci-1cn-1fe, protobuf schema registry"
+RUST_LOG="info,risingwave_stream=info,risingwave_batch=info,risingwave_storage=info" \
+cargo make ci-start ci-1cn-1fe &&
+python3 -m pip install psycopg2-binary, protobuf, confluent-kafka && \
+python3 e2e_test/schema_registry/pb.py && \
```

Comment:
```suggestion
cargo make ci-start ci-1cn-1fe
python3 -m pip install psycopg2-binary, protobuf, confluent-kafka
python3 e2e_test/schema_registry/pb.py
```

`set -e` at the start of the script will cause it to exit on error.